src/mstore.erl

```diff
@@ -390,14 +391,6 @@ do_put(MStore = #mstore{size=S, data_size = DataSize}, Metric,
     {ok, F1} = write(F, DataSize, Metric, Time, Data),
     do_put(MStore, Metric, R, DataRest, [{FileBase, F1} | FileRest]);
 
-do_put(MStore = #mstore{size=S, data_size = DataSize}, Metric,
-       [{Time, Size} | R], InData,
-       [First, {FileBase, F}]) when
-      ((Time div S)*S) =:= FileBase ->
```

Comment:
You are right, the source of the Time Value is the head of the list in the former function and is the second entry in the list in the "deleted" version. I missed that originally, and will add it back.